lms/djangoapps/courseware/views/views.py

```diff
@@ -726,6 +726,12 @@ def _progress(request, course_key, student_id):
 
     # checking certificate generation configuration
     enrollment_mode, is_active = CourseEnrollment.enrollment_mode_for_user(student, course_key)
+
+    # We need to have id verified to show generate cert button
+    # in the case where courseMode is of Verified variety.
+    is_id_verified = SoftwareSecurePhotoVerification.user_is_verified(student) or \
+        enrollment_mode and not CourseMode.has_verified_mode(enrollment_mode)
```

Comment:
I think there will be some scenarios where we want to show the cert button even if a course _has_ a verified mode. (Like if a course has a verified, audit, and honor track.)

Also, we shouldn't name this `is_id_verified` since someone in an honor code track probably won't be verified.